### PR TITLE
OCLOMRS-648: Add tests to increase the test coverage

### PR DESCRIPTION
**Pull request by danuluma**
# JIRA TICKET NAME:
[Add tests to increase the test coverage](https://issues.openmrs.org/browse/OCLOMRS-648)

# Summary:
Add tests to increase the test coverage to address the drop observed in #502

### CODE REVIEW

File: src/tests/Dashboard/reducer/conceptReducer.test.js [hunk 1]
```diff
@@ -13,8 +13,10 @@ import {
   ADD_NEW_SET_ROW,
   REMOVE_SELECTED_SET,
   ADD_SELECTED_SETS,
-  PRE_POPULATE_SETS, UNPOPULATE_PRE_POPULATED_SETS,
+  PRE_POPULATE_SETS,
```
Comment (dkayiwa) [inline src/tests/Dashboard/reducer/conceptReducer.test.js]: Is this related to increasing the test coverage?

Comment (danuluma) [reply src/tests/Dashboard/reducer/conceptReducer.test.js]: Its a refactor of the imports for consistency and readability- each on its own line.
